<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MVHub</class>
 <widget class="QMainWindow" name="MVHub">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MVHub</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icon.png</normaloff>:/Icons/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="update_consent">
      <property name="text">
       <string>update consent data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="api_redcap_se">
      <property name="text">
       <string>RedCap (SE)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="api_redcap_case">
      <property name="text">
       <string>RedCap (Case Management)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="api_pseudo">
      <property name="text">
       <string>API Pseudonymisierung</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="api_redcap_se_upload">
      <property name="text">
       <string>RedCap (SE) Upload</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="table">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>PS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>additional samples</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>SAP ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CM ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Netzwerk</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Consent</string>
        </property>
       </column>
      </widget>
      <widget class="QPlainTextEdit" name="output"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
